frontend/src/lib/ocp-utils.ts

```diff
@@ -9,20 +11,27 @@ function api<T>(url: string, headers?: Record<string, unknown>): Promise<T> {
 }
 
 export function launchToOCP(urlSuffix: string, newTab: boolean, onError?: VoidFunction) {
-    api<{ data: { consoleURL: string } }>(
-        '/multicloud/api/v1/namespaces/openshift-config-managed/configmaps/console-public/'
-    )
-        .then(({ data }) => {
+    const resourceResult = getResource({
+        apiVersion: ConfigMapApiVersion,
+        kind: ConfigMapKind,
+        metadata: {
+            name: 'console-public',
+            namespace: 'openshift-config-managed',
+        },
+    }).promise
```

Comment:
@fxiang1 Since we only run inside the OpenShift console, you don't need to look up this URL. You can just use a link starting with /

I have removed some of these cases, but some still remain. The only thing is that it won't work for local development, but I think we should probably come up with something that is not used in production for that case.

Reply:
Thanks @KevinFCormier ! I changed it to just use /. For local development I agree we should come up with something.